library-best-practices/JLBP-17.md

```diff
@@ -0,0 +1,30 @@
+[JLBP-17] Coordinate Major Version Adoption
+-------------------------------------------
+
+It is clearly undesirable for any particular library dependency tree to contain
+conflicts. Unfortunately, for any consumer of such a dependency tree that tries
```

Comment:
"Unfortunately, for any consumer of such a dependency tree that tries
to use the highest versions available of all libraries contained in it, there
are conflicts as soon as a breaking change is introduced."

I'm not sure I follow this sentence.

Reply:
I will rewrite it.